Danki2/Assets/Scripts/Actor/Mortal/Enemy/TargetDummy.cs

```diff
@@ -19,7 +19,7 @@ protected override void Awake()
 
     protected override void OnDeath()
     {
-        // TODO: Implement TargetDummy death.
-        Debug.Log("The target dummy died.");
+        gameObject.GetComponent<Rigidbody>().velocity = Vector3.up * 5;
```

Comment:
We're already getting the rigid body in the actors `Start` method, I know you said you didn't want to make this protected so that anyone could use it, maybe the movement manager should expose methods for doing things like this? Just something to think about as I don't think it should block this going in, happy to merge this branch as is (once the parent branch is in)

Reply:
Yeah that's definitely food for thought. I reckon part of the reason we use it here is at the point of death we don't care about overriding the movement-manager.